siva/library.go

```diff
@@ -75,6 +78,10 @@ func NewLibrary(
 		return nil, err
 	}
 
+	if ops.Transactional && ops.RegistryCache <= 0 {
```

Comment:
Do it also for non transactional. It also affects the cache of references and config.

Reply:
Also change documentation of `RegistryCache` from the options to state that 0 does not disable it but sets it to a default value.